internal/usecase/translation_test.go

```diff
@@ -114,6 +121,8 @@ func TestTranslate(t *testing.T) {
 
 		t.Run(tc.name, func(t *testing.T) {
 			t.Parallel()
```

Comment:
```suggestion
			// t.Parallel()
```
Also, we could just delete this line